core/vm/instructions_test.go

```diff
@@ -927,3 +927,50 @@ func TestOpMCopy(t *testing.T) {
 		}
 	}
 }
+func TestOpSetIndestructible(t *testing.T) {
```

Comment:
Agree with @jmadibekov the way tests are written now - they are not testing much. Basically they are testing if Go's `map` works correctly 😅
I would urge you to look at the test @jmadibekov  added to his PR (you don't have to add fuzz tests), but I would suggest _stronger_ test here, fell free to ask @jmadibekov  for help :)